### PR TITLE
chore: bump ci dependencies

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or integration tests where appropriate
- Backwards compatible: Works with existing data and configuration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
This gets rid of those concerning build deprecation warnings. I've also replaced the 3rd party pip action with the native installer which feels more reliable.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,24 +12,20 @@ jobs:
...
-    - name: Installing wheel package for pip # This makes the warning about legacy installing go away.
-      uses: BSFishy/pip-action@v1
-      with:
-        packages: wheel
-    - name: Installing pyxform # Install pyxform dependency.
-      uses: BSFishy/pip-action@v1
-      with:
-        packages: git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic
+    - name: Install wheel
+      run: pip install wheel
```
Comment (jkuester) [inline .github/workflows/build.yml:26]: Slightly unrelated, but why do we need `wheel`?  I read through [the documentation](https://pip.pypa.io/en/stable/cli/pip_wheel/) and I am still not sure I understand exactly what it does (but then I don't have too much experience with Python....).  Either way, I don't think I ever installed this on my local machine before using pyxform, so I am not sure why we need it here....

**Review by jkuester — APPROVED** [review]
:+1:

File: .github/workflows/build.yml (hunk 1 above)
Comment (garethbowen) [reply .github/workflows/build.yml:26]: No idea. I'll merge this but let me know if you think it's worth raising a new issue to investigate...

**Pull request merged**